src/main/java/org/jboss/snowdrop/licenses/xml/DependencyElement.java

```diff
@@ -161,4 +166,19 @@ public int hashCode() {
         result = 31 * result + version.hashCode();
         return result;
     }
+
+    public String toGav() {
+        return String.format("%s:%s:%s", groupId, artifactId, version);
+    }
+
+    private String toCommunityGav() {
```

Comment:
Not needed as commented in ExceptionLicenseSanitiser

Reply:
Removed